src/dynamics/revolute_joint.rs

```diff
@@ -138,6 +145,38 @@ impl RevoluteJoint {
         self.data.set_limits(JointAxis::AngX, limits);
         self
     }
+
+    /// The angle along the free degree of freedom of this revolute joint in `[-π, π]`.
```

Comment:
```suggestion
    /// The angle along the free degree of freedom of this revolute joint in `[-π, π]`.
    ///
    /// See also [`Self::angle`] for a version of this method taking entities instead of rigid-body handles.
    /// Similarly [`RapierContext::impulse_revolute_joint_angle`] only takes a single entity as argument to compute that angle.
```